tests/transformers/searchservice/createDirectAnswer.ts

```diff
@@ -59,4 +61,42 @@ it('can create a FieldValueDirectAnswer', () => {
     fieldType: 'phone'
   };
   expect(actualDirectAnswer).toMatchObject(expectedDirectAnswer);
+});
+
+it('can create a custom FieldValueDirectAnswer', () => {
+  const apiFieldValueDirectAnswer = {
+    type: 'FIELD_VALUE',
+    answer: {
+      value: {
+        color: 'yellow',
+        os: 'android'
+      },
+      entityName: 'Obama Phone',
+      fieldName: 'Phone Description',
+      fieldApiName: 'phoneDescription',
+      fieldType: 'c010101.specialtype'
```

Comment:
is it ok that we're replacing the value for a user's custom `fieldType` with 'unknown'? I'm wondering if it might be information that users want to be able to access somehow so they can differentiate between their custom fields

Reply:
product is ok with it. User can access `fieldApiName` to check for their specific custom field and assert the typing from there!

Reply:
oh ok, `fieldApiName` is another unique string that they define when creating a custom field?

Reply:
fieldApiName is the field name within an entity schema -- the fields you see generated when using the CLI command. User would provide a list of those field names (fieldApiName) to their experience to configure what can be use as a direct answer